prime-router/src/main/resources/db/migration/V67__update_list_send_failure.sql

```diff
@@ -0,0 +1,90 @@
+/*
+* The Flyway tool applies this migration to create the database.
+*
+* Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+* use VARCHAR(63) for names in organization and schema
+*
+* Copy a version of this comment into the next migration
+*
+*/
+
+/*
+    This is the "Last Mile" failures query.
+    See https://github.com/CDCgov/prime-reportstream/issues/5771 ,
+        https://github.com/CDCgov/prime-reportstream/issues/9649 ,
+        https://github.com/CDCgov/prime-reportstream/issues/13073
+ */
+DROP FUNCTION IF EXISTS list_send_failures;
+CREATE OR REPLACE FUNCTION list_send_failures(days_to_show_param INT)
+    RETURNS TABLE
+            (
+                action_id            BIGINT,
+                report_id            UUID,
+                receiver             TEXT,
+                file_name            TEXT,
+                failed_at           TIMESTAMP WITH TIME ZONE,
+                action_params        varchar(2048),
+                action_result        TEXT,
+                body_url             varchar(2048),
+                report_file_receiver TEXT
+            )
+    LANGUAGE plpgsql
+AS
+$$
+
+BEGIN
+    RETURN QUERY
+        WITH send_failures as
+                 (select a.action_id
+                       , (a.action_params::jsonb->>'reportId')::uuid
```

Comment:
This is the actual change, changing from: `split_part(a.action_params, '&', 3)::uuid`